maven/bnd-maven-plugin/pom.xml

```diff
@@ -26,6 +26,12 @@
 			<groupId>org.apache.maven</groupId>
 			<artifactId>maven-compat</artifactId>
 		</dependency>
+		<dependency>
+			<groupId>org.sonatype.plexus</groupId>
+			<artifactId>plexus-build-api</artifactId>
+			<version>0.0.7</version>
```

Comment:
I just realized this version needs to go in the parent pom. We manage all the versions there. See https://github.com/bndtools/bnd/blob/master/maven/pom.xml#L53 for the dependencies. Since the version will be set by the parent pom, you can remove the version here.